application/libraries/Api/Rest/V1/SchemaFactory/SchemaFactoryPersonalSettings.php

```
@@ -0,0 +1,19 @@
+<?php
+
+namespace LimeSurvey\Api\Rest\V1\SchemaFactory;
+
+use GoldSpecDigital\ObjectOrientedOAS\Objects\Schema;
+use GoldSpecDigital\ObjectOrientedOAS\Contracts\SchemaContract;
+
+class SchemaFactoryPersonalSettings
+{
+    public function make(SchemaContract ...$properties): Schema
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$properties'. (undefined)

(UnusedFormalParameter)